plugin/trino-hive/src/main/java/io/trino/plugin/hive/orc/OrcTypeTranslator.java

```diff
@@ -14,15 +14,24 @@
 package io.trino.plugin.hive.orc;
 
 import io.trino.orc.metadata.OrcType.OrcTypeKind;
+import io.trino.plugin.hive.coercions.IntegerNumberToVarcharCoercer;
```

Comment:
Can we reowrd the commit message as `Add support for integer number to varchar coercer in unpartitioned table` and in the commit description we could mention them in detail

Reply:
Done